pkg/shell/privileges.js

```diff
@@ -60,6 +60,10 @@ function Privileges() {
         display();
     };
 
+    self.is_locked = function is_locked() {
+        return locked;
+    };
```

Comment:
This isn't being used any more, so that can go. But we still need a replacement for "cockpit-ws has a password for the user"